### DISCUSSION TITLE
Allowing everyone to access chests in claim

### DISCUSSION THREAD

**Discussion by YueMiyuki** [GriefPrevention/GriefPrevention#2069] [Q&A] [answered]
Hi. I am trying to allow everyone to access chests in claim, but not hoppers or furnaces, etc.
I tried `/containertrust` but that doesnt do the trick. Does anyone know how I can achieve this? thanks

**Comment by bigpresh** [comment]
The containertrust permissions affect anything with an inventory, e.g. chests, furnaces etc - unfortunately you can't grant permission to only a certain type of container-like block.

You can place your chests (and only the chests) within a subdivision of the claim and grant containertrust on just that subdivision, not the entire claim, I think that's about the best way you can do it.

**Reply by YueMiyuki** [reply]
But how do I make it accessable to everyone?

**Reply by bigpresh** [reply, marked as answer]
Oh - `/containertrust public` - I thought your question was on how to limit to types of things, not how to give that access to everyone so I omitted that bit!

**Reply by YueMiyuki** [reply]
Oh, I see. And how do I make a subregion please?

**Reply by Jikoo** [reply]
`/subdivideclaims` to enter subdivision mode, switch off the shovel to swap back out.

**Reply by YueMiyuki** [reply]
Thanks. But now my water is not flowing in the subregion claim, how can I fix that?
![image](https://github.com/TechFortress/GriefPrevention/assets/76854136/0de4b12c-11d6-4d71-821c-c4cdd047acbf)

**Reply by Jikoo** [reply]
Update to the current beta release that changes water flow restrictions or move your storage to a location not directly beneath the platforms.

**Reply by YueMiyuki** [reply]
I updated to the current beta but its the same thing

**Reply by Jikoo** [reply]
Did you update the existing water?

**Reply by YueMiyuki** [reply]
I did, I put the water away and put it back, but its still like that

**Reply by Jikoo** [reply]
So for clarity, you are on 16.18.2-beta-1 and water is not flowing into a subclaim that is not restricted? If that is the case, please open a bug report with the relevant information filled out.

As for workarounds, either place the water prior to creating the subclaim or move the subclaim and corresponding storage to a place where they will not interfere with the water above.

**Reply by YueMiyuki** [reply]
Alright, I have opened issue #2079
